core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -54,7 +56,7 @@ import org.apache.spark.util.{AkkaUtils, Utils}
 @DeveloperApi
 class SparkEnv (
     val executorId: String,
-    val actorSystem: ActorSystem,
+    val rpcEnv: RpcEnv,
```

Comment:
Oh, and if code outside of Spark is not supposed to use `rpcEnv`, it also should be `private[spark]`.